xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -1377,6 +1393,20 @@ struct AppModel: ModelProtocol {
                 ),
                 environment: environment
             )
+        case let .failUpdateLikeStatus(address, error):
+            logger.warning(
+                """
+                Failed to update like status entry: \(address)
+                \(error)
+                """
+            )
+            return update(
+                state: state,
+                action: .pushToast(
+                    message: "Could not update status"
+                ),
+                environment: environment
+            )
```

Comment:
Nit: break out into function. I do this reflexively now, due to our earlier problems with the Swift compiler seeming to choke on large switch statements with many statements inside the arms.